src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1014,7 +1011,7 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
       }
     }
   }
-
+  
```

Comment:
It looks like the only thing it would roll back is setting the Workflow status to Aborting.  I think that is an accurate status even if we have failed to do so.  What do others think?

If we keep this ErrorReport non-Exceptional, we should add a comment explaining our logic.